src/extension/src/signalr-api-module/generateCommand.ts

```diff
@@ -17,7 +17,8 @@ export class GenerateCommand extends CoreTemplateStudioApiCommand {
 
     connection.on(CONSTANTS.API.GEN_LIVE_MESSAGE_TRIGGER_NAME, genMessage => {
       generatedItemsCount++;
-      const messageWithProgress = `(${generatedItemsCount}/${itemsToGenerateCount}) ${genMessage}`;
+      const percentage = (generatedItemsCount / itemsToGenerateCount) * 100;
+      const messageWithProgress = `(${percentage.toFixed(0)} %) ${genMessage}`;
```

Comment:
NIT: you should remove the space before the `%` sign.
I've never seen it used that way.
i.e. it should be `25%` and not `25 %`

Reply:
Oops, thanks! :)

Reply:
Fixed